apps/kubernetes-provider/src/index.ts

```diff
@@ -417,7 +417,7 @@ class KubernetesTaskOperations implements TaskOperations {
     type: THookType,
     cause: THookType extends "postStart" ? PostStartCauses : PreStopCauses
   ) {
-    return ["/bin/sh", "-c", `sleep 1; wget -q -O- 127.0.0.1:8000/${type}?cause=${cause}`];
+    return ["/bin/sh", "-c", `for i in $(seq 1 5); do sleep 1; wget -q -O- 127.0.0.1:8000/${type}?cause=${cause} && break; done`];
```

Comment:
For things like this which are a bit hard to read (if you're not awesome at the command line) it'd be good to have a code comment on the line above saying what it does.